frodo.js

```diff
@@ -6,73 +6,75 @@
  *
```

Comment:
SCRIPT-There seems to be a typo in the function name. Did you mean `calculateAverage` instead of `calculateAverge`?